### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by AKBuggy**
I am a newbie and wanted to contribute to an open-source project. So, I started updating README.md. Fixed some grammatical errors and sentences.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # privateGPT
-Ask questions to your documents without an internet connection, using the power of LLMs. 100% private, no data leaves your execution environment at any point. You can ingest documents and ask questions without an internet connection!
+Ask questions about your documents without an internet connection, using the power of LLMs. 100% private; no data leaves your execution environment at any point. You can ingest documents and ask questions without an internet connection!
```
Comment (AKBuggy) [inline README.md:2]: Please check if this needs to changed back to "to"

File: README.md [hunk 2]
```diff
@@ -63,4 +63,4 @@ Selecting the right local models and the power of `LangChain` you can run the en
 - `GPT4All-J` wrapper was introduced in LangChain 0.0.162.
 
 # Disclaimer
-This is a test project to validate the feasibility of a fully private solution for question answering using LLMs and Vector embeddings. It is not production ready, and it is not meant to be used in production. The models selection is not optimized for performance, but for privacy; but it is possible to use different models and vectorstores to improve performance.
+This is a test project to validate the feasibility of a fully private solution for question answering using LLMs and vector embeddings. It is not production-ready, and it is not meant to be used in production. The model selection is not optimized for performance but for privacy, but it is possible to use different models and vector stores to improve performance.
```
Comment (AKBuggy) [inline README.md:66]: Please check whether I should change this back to vectorstores

File: README.md (hunk 1 above)
Comment (johanwulf) [reply README.md:2]: I think "to" in this context is that the LLM assumes your document as a "character" so you are asking questions to your document

Comment (AKBuggy) [reply README.md:2]: So, I'll make some quick changes.

**Pull request closed without merging**